src/Tizen.NUI.Wearable/src/public/CircularSlider.cs

```diff
@@ -0,0 +1,674 @@
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.thumbSize = (Size)newValue;
+                instance.UpdateThumbVisualSize((Size)newValue);
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.thumbSize;
+        });
+
+        /// <summary>Bindable property of ThumbColor</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ThumbColorProperty = BindableProperty.Create("thumbColor", typeof(Color), typeof(CircularSlider), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+
+            // TODO : Set viewStyle.ThumbColor after style refactoring done.
+
+            instance.UpdateThumbVisualColor((Color)newValue);
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            return ((CircularSliderStyle)((CircularSlider)bindable).viewStyle)?.ThumbColor;
+        });
+
+        /// <summary>Bindable property of IsEnabled</summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty IsEnabledProperty = BindableProperty.Create(nameof(IsEnabled), typeof(bool), typeof(CircularSlider), true, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (CircularSlider)bindable;
+            if (newValue != null)
+            {
+                instance.privateIsEnabled = (bool)newValue;
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (CircularSlider)bindable;
+            return instance.privateIsEnabled;
+        });
+
+        private static readonly string TrackVisualName = "Track";
+        private static readonly string ProgressVisualName = "Progress";
+        private static readonly string ThumbVisualName = "Thumb";
+        private ArcVisual trackVisual;
+        private ArcVisual progressVisual;
+        private ArcVisual thumbVisual;
+
+        private float maxValue = 100;
+        private float minValue = 0;
+        private float currentValue = 0;
+        private Size thumbSize;
+        private bool isEnabled = true;
+
+        private EventHandler<ValueChangedArgs> valueChangedHandler;
+
+        float sliderPadding = 6.0f;
+
+        private Animation sweepAngleAnimation;
+        private Animation thumbAnimation;
+
+        /// <summary>
+        /// Get style of progress.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new CircularSliderStyle Style => ViewStyle as CircularSliderStyle;
+
+        /// <summary>
+        /// Value Changed event data.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public class ValueChangedArgs : EventArgs
```

Comment:
Use `EventArgs` sufffix instead of `Args`(StyleCop rule violation(ca1710)).

https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1710?view=vs-2019